extruct/uniform.py

```diff
@@ -20,10 +20,21 @@ def _umicrodata_microformat(extracted, schema_context):
             res.append(flatten_dict(obj, schema_context, True))
     elif isinstance(extracted, dict):
         res.append(flatten_dict(extracted, schema_context, False))
-
     return res
 
 
+def flatten(element, schema_context):
```

Comment:
Hey! Would you mind making this method private (_flatten), as it is not something users should call.